### PR TITLE
There are different endpoints for eu and us sites

### PR DESCRIPTION
**Pull request by tapulskis**
adds eu sites to the guide

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by drichards-87** [comment]
Thanks for submitting this PR. I've created DOCS-4970 for review.

File: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Before you begin, you need a [Datadog account][2], with [an API key][3], and you
 
 1. Follow the steps in the [Amazon Create an API destination docs][5] to add Datadog as an API destination.
     - Use API key authorization, with `DD-API-KEY` as your key name and your [Datadog API key][3] as the value.
-    - For your destination endpoint, use `https://http-intake.logs.datadoghq.com/v1/input` for logs or `https://api.datadoghq.com/api/v1/events` for events, and set `POST` as the HTTP method. For more information about the differences between logs and events, see the [logs section][6], and the [events section][7] of the [Categories of Data docs page][8].
+    - For your destination endpoint, use `https://http-intake.logs.datadoghq.com/v1/input` or `https://http-intake.logs.datadoghq.eu/v1/input`   for logs and `https://api.datadoghq.com/api/v1/events` or `https://api.datadoghq.eu/api/v1/events` for events, and set `POST` as the HTTP method. For more information about the differences between logs and events, see the [logs section][6], and the [events section][7] of the [Categories of Data docs page][8].
```
Comment (maycmlee) [inline content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md:20]:
```suggestion
    - For your destination endpoint, use `https://{{< region-param key="http_endpoint" code="true" >}}/v1/input` for logs and `https://api.{{< region-param key="dd_site" code="true" >}}/api/v1/events` for events, and set `POST` as the HTTP method. For more information about the differences between logs and events, see the [logs section][6], and the [events section][7] of the [Categories of Data docs page][8].
```

**Review by maycmlee — COMMENTED** [review]
Thanks @tapulskis for opening this PR! I made some edits so that the URLs uses our region shortcode, so when change the site, the URL will update to the correct URL endpoint for that site.

**Review by tapulskis — COMMENTED** [review]
Thank you for the edit I was unaware of these special variables :)

**Comment by maycmlee** [comment]
Thanks again @tapulskis! And we don't expect you to know about the special variables :)

**Pull request merged**